pkg/operator/controller/controller_test.go

```diff
@@ -160,6 +161,8 @@ var _ = Describe("Controller", func() {
 				Expect(conditions.IsStatusConditionFalse(args.cdi.Status.Conditions, conditions.ConditionDegraded)).To(BeTrue())
 
 				Expect(args.cdi.Finalizers).Should(HaveLen(1))
+
+				validateEvents(args.reconciler.recorder.(*record.FakeRecorder).Events, createReadyEventValidationMap())
```

Comment:
maybe change the signature of validateEvents to take args or args.reconciler to avoid the ugly cast in every invocation?

Reply:
Done